src/plugins/vis_type_timeline/server/series_functions/graphite.test.js

```diff
@@ -34,7 +34,16 @@ const expect = require('chai').expect;
 
 import fn from './graphite';
 
```

Comment:
We should have a couple of unit tests that cover the case where both the allowlist and blocklist are enabled if we don't have it already

Reply:
will do